docs/CONFIG-VARS.md

```diff
@@ -7,7 +7,6 @@ Supported configuration variables are listed in the table below.  All variables
 - [List of valid configuration variables](#list-of-valid-configuration-variables)
   - [Table of Contents](#table-of-contents)
   - [Required Variables](#required-variables)
-    - [Application](#application)
```

Comment:
Is there a reason as to why the link to "Application" was removed?

Reply:
Yes, to keep it consistent with the AWS repo. If it's need we need to make changes across all providers.